<launch>
    <!-- launch specification file to get parameters for the task -->
    <include file="$(find uniform_object_rearrangement)/launch/task_specification.launch"></include>

    <!-- monotone experiments setting up -->
    <!-- arg (1)#experiment per object (2)#instances needed per object (3)time_allowed-->
    <node pkg="uniform_object_rearrangement" type="MonotoneExperiments.py" name="monotone_experiments"
        args="$(arg monotone_experiments)" output="screen" required="True" />

    <node pkg="uniform_object_rearrangement" type="PybulletExecutionScene.py" name="pybullet_execution_scene"
		    output="screen" required="True" />

    <node pkg="uniform_object_rearrangement" type="PybulletPlanScene.py" name="pybullet_plan_scene"
		    output="screen" required="True" />

    <node pkg="uniform_object_rearrangement" type="PoseEstimator.py" name="fake_pose_estimator"
        output="screen" required="False" />

    <node pkg="uniform_object_rearrangement" type="main_planner_node" name="main_planner" output="screen" />
   
</launch>